yalafi/parameters.py

```diff
@@ -393,9 +393,9 @@ def init_math_collections(self):
 
             '+', '-', '\\cdot', '\\times', '/',
             '=', '\\eq', '\\ne', '\\neq',
-            '<', '>', '\\le', '\\leq', '\\ge', '\\geq',
-            ':', ':=', '\\to', '\\cap', '\\cup',
-            '\\Leftrightarrow',
+            '<', '>', '≤', '≥', '\\le', '\\leq', '\\ge', '\\geq',
+            ':', ':=', '→', '\\to', '∩', '\\cap', '∪', '\\cup', '⇒', '\\Rightarrow', '⇐', '\\Leftarrow',
+            '⇔', '\\Leftrightarrow', '⊂',

```

Comment:
Is it okay to add those globally? So they apply even if `unicode-math` is not used.

Reply:
Maybe, you can extend the list in `init_module()`, similarly to the activation of macro `\text` in yalafi/packages/amsmath.py?

Reply:
Sure. Since I also added `'\\Rightarrow'`, shall this stay there?

Reply:
Oh, yes. I didn't scroll the above code snipped to the right, so missed your \\Leftarrow and \\Rightarrow. These should best stay in the global list.